cmd/k3d/destroy.go

```diff
@@ -223,9 +223,9 @@ func destroyK3d(_ *cobra.Command, _ []string) error {
 		viper.WriteConfig()
 	}
 
-	if _, err := os.Stat(config.K1Dir + "/kubeconfig"); !os.IsNotExist(err) {
-		if err := os.Remove(config.K1Dir + "/kubeconfig"); err != nil {
-			return fmt.Errorf("unable to delete %q: %w", config.K1Dir+"/kubeconfig", err)
+	if _, err := os.Stat(config.K1Dir); !os.IsNotExist(err) {
```

Comment:
i understand the desire to remove the k1dir. i want to also question that this is a change instead of an addition. why don't we want to remove the kubeconfig any longer?

Reply:
the ultimate goal to make a k3d provisioning possible again is to to remove the directory,i am not sure what was the thought behind removing kubeconfig,but this change should work smoothly for k3d reprovisioning